packages/salesforcedx-sobjects-faux-generator/src/messages/i18n.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * Conventions:
+ * _message: is for unformatted text that will be shown as-is to
+ * the user.
+ * _text: is for text that will appear in the UI, possibly with
+ * decorations, e.g., $(x) uses the https://octicons.github.com/ and should not
+ * be localized
+ *
+ * If ommitted, we will assume _message.
+ */
+export const messages = {
+  faux_generation_cancelled_text: 'Faux class generation cancelled',
+  failure_fetching_sobjects_list_text:
+    'Failure fetching list of SObjects from org %s',
+  failure_in_sobject_describe_text: 'Failure performing SObject describe %s',
+  no_sobject_output_folder_text: 'No output folder available %s',
+  fetched_sobjects_length_text:
+    'Fetched %s %s SObjects from default scratch org\n',
+  no_generate_if_not_in_project:
+    'Unable to generate faux classes for SObjects when not in an SFDX project %s'
```

Comment:
The 's' in "sObject" should be lowercase when the word is used in a sentence like this. Also, can we give the user more info on how to move forward? Or is `%s` already appending that?

Reply:
I think this is about all that we can tell them.   It just means that somehow they aren't doing all this in an SFDX project - opened vscode in the wrong place.    Hopefully, we are checking this for most all operations in the CLI and VSCode, but I haven't confirmed that.